README.md

```diff
@@ -5,16 +5,35 @@ Those credentials will be valid for 12 hours by default.
 
 ## Getting started
 
+### Install using go...
+
+For this, go must be installed on your system. 
+
 Install executable with golang
 ```bash
-go get [-u] github.com/signavio/aws-mfa-login
+go get github.com/signavio/aws-mfa-login
 ```
-or download from releases
-```yaml
+.. or update by setting the -u flag: 
+```bash
+go get -u github.com/signavio/aws-mfa-login
+``` 
+
+Make sure your go path is part of your PATH environment variable: 
+```
+export PATH="${PATH}:~/go/bin/"
```

Comment:
add also export GOPATH=~/go because this might depend on which install method you used.

Reply:
done